### PR TITLE
KUP-32 upload event image

### PR DESCRIPTION
**Pull request by codecventer**
- Upload Image to S3
- In process of storing image URL to Dynamo DB

### CODE REVIEW

File: Frontend/components/dashboard/EventImage.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    const uploadParams = {
+      Bucket: S3_BUCKET_NAME,
+      Key: "image-name",
+      Body: file,
+      Tagging: "Event ID=0123456789",
+    };
+
+    try {
+      const upload = await s3.send(new PutObjectCommand(uploadParams));
+      const object = `${S3_URL}${uploadParams.Key}`;
```
Comment (Sjk-Lacoste) [inline Frontend/components/dashboard/EventImage.js:87]: Where are you saving this?

**Review by Sjk-Lacoste — CHANGES_REQUESTED** [review]

File: Frontend/components/dashboard/EventImage.js (hunk 1 above)
Comment (codecventer) [reply Frontend/components/dashboard/EventImage.js:87]: I use it to construct an Object URL to store to the DB

Comment (codecventer) [reply Frontend/components/dashboard/EventImage.js:87]: But it's only console logged at the moment

Comment (Sjk-Lacoste) [reply Frontend/components/dashboard/EventImage.js:87]: May you please set it to a hook that can be used on the save event function

**Pull request merged**